README.md

```diff
@@ -115,37 +161,51 @@ app.mount('#app');
 
 ## Using the SDK
 
-Once FullStory is initialized, you can make calls to the FullStory SDK.
+Once FullStory is initialized, you can make calls to the FullStory SDK. See the [developer documentation](https://developer.fullstory.com/browser/v2/getting-started/) for more information.
 
 ### Sending custom events
 
 ```JavaScript
-FullStory.event('Subscribed', {
-  uid_str: '750948353',
-  plan_name_str: 'Professional',
-  plan_price_real: 299,
-  plan_users_int: 10,
-  days_in_trial_int: 42,
-  feature_packs: ['MAPS', 'DEV', 'DATA'],
+FullStory('trackEvent', {
+  name: 'Subscribed',
+  properties: {
+    uid: '750948353',
+    plan_name: 'Professional',
+    plan_price: 299,
+    plan_users: 10,
+    days_in_trial: 42,
+    feature_packs: ['MAPS', 'DEV', 'DATA'],
+  },
+  schema: {
+    properties: {
+      plan_users: 'int',
+      days_in_trial: 'int',
+    }
+  }
```

Comment:
to match the v1 call I'm updating, we need this schema. I could also simplify and remove the schema.

Reply:
I think having the schema declaration is cool. I feel you should add the comment for clarity + to be consistent with the page props code example:

```
schema: {
    properties: {
      plan_users: 'int', // override default inferred "real" type
      days_in_trial: 'int', // override default inferred "real" type
    }
```

Reply:
I tend to agree! I might find a suitable place to link to the inference page in dev docs as well.